you-should-use.plugin.zsh

```diff
@@ -111,7 +127,6 @@ You should use: ${PURPLE}\"%alias\"${NONE}"
     _write_ysu_buffer "$MESSAGE\n"
```

Comment:
`_write_ysu_buffer` prints an empty line when `$MESSAGE` is empty

```
[[ -n $MESSAGE ]] && _write_ysu_buffer "$MESSAGE\n"
```
this should go here